lib/floe/cli.rb

```diff
@@ -25,11 +26,25 @@ def run(args = ARGV)
           Floe::Workflow.load(workflow, context)
         end
 
-      Floe::Workflow.wait(workflows, &:run_nonblock)
+      output_streams =
+        if workflows.size == 1 || !opts[:segment_output]
+          # no extra work necessary
+          {}
+        else
+          workflows.each_with_object({}) do |workflow, h|
+            workflow.context.logger = Logger.new(output = StringIO.new)
```

Comment:
I'm not sure I like this.  I like seeing the live logs when using exe/floe, because I can see what state its in as it's running.

Reply:
Oh wait, I misread the code.  I see that passing segment output will trigger this section otherwise it works normally. 👍